geode-core/src/main/kotlin/org/apache/geode/cache/rest/controllers/RegionsController.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.geode.cache.rest.controllers
+
+import org.apache.geode.cache.configuration.RegionAttributesDataPolicy
+import org.apache.geode.cache.configuration.RegionAttributesType
+import org.apache.geode.cache.configuration.RegionConfig
+import org.apache.geode.internal.cache.GemFireCacheImpl
+import org.apache.geode.management.internal.cli.CliUtil
+import org.apache.geode.management.internal.cli.functions.CliFunctionResult
+import org.apache.geode.management.internal.cli.functions.CreateRegionFunctionArgs
+import org.apache.geode.management.internal.cli.functions.RegionCreateFunction
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+
+data class RegionCreateParams @JvmOverloads constructor(val name: String = "", val type: String = "")
+
+@RestController
+class RegionsController {
+
+    @RequestMapping(method = [RequestMethod.POST], value = ["/regions"],
+            produces = ["application/json"], consumes = ["application/json"])
+    fun create(@ModelAttribute params: RegionCreateParams): ResponseEntity<String> {
+        var regionConfig = RegionConfig()
```

Comment:
This can be a `val`. So it _should_ be a `val`.

I think every single `var` in this PR so far should be a `val`.

Reply:
Fair point!